### DISCUSSION TITLE
About ' :e ' used in Idea

### DISCUSSION THREAD

**Discussion by blueair5** [JetBrains/ideavim#650] [Q&A]
When I want to use `:e` to open a file, I want to use tab for completion of the filename。But it doesn't work. Anyone has answers about this. Thanks.

**Comment by AlexPl292** [comment]
Hi, please track this issue: [VIM-268](https://youtrack.jetbrains.com/issue/VIM-268)

**Reply by blueair5** [reply]
Ok, Thanks
